package.json

```diff
@@ -51,7 +51,6 @@
     "prettier": "^2.5.1"
   },
   "dependencies": {
-    "doctoc": "^1.4.0",
```

Comment:
didnt really need to remove this as its a dev only dependency for organising README contents. But removed it anyway to remove a high severity issue